backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/IDocTypeBL.java

```diff
@@ -1,15 +1,24 @@
 package de.metas.document;
 
+import de.metas.document.invoicingpool.DocTypeInvoicingPoolId;
 import de.metas.i18n.ITranslatableString;
 import de.metas.util.ISingletonService;
+import lombok.NonNull;
 import org.compiere.model.I_C_DocType;
 
+import java.util.Set;
+
 public interface IDocTypeBL extends ISingletonService
 {
 	I_C_DocType getById(DocTypeId docTypeId);
+	
+	I_C_DocType getByIdNonNull(DocTypeId docTypeId);
```

Comment:
can u pls unify with getById(),
make sure getById returns not null
check where getById is called and see if null is expected.... shall not be